__tests__/__unit__/Profiles.unit.test.ts

```diff
@@ -163,7 +196,6 @@ describe("Profile class unit tests", () => {
             createInputBox.mockReturnValue(inputBox);
             profiles.getUrl = () => new Promise((resolve) => { resolve(undefined); });
             await profiles.createNewConnection(profileOne.name);
-            expect(showInformationMessage.mock.calls.length).toBe(1);
```

Comment:
The length is more than one. Since Stepan said[ this](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Unit-Tests#each-unit-test-should-have-a-clear-purpose), I decided to remove it.

Reply:
Makes sense. In that case, maybe having a test for `toBeGreaterThanOrEqual(1)` would explain that to the reader. @stepanzharychevbroadcom what's your take on this?